### ISSUE TITLE
Client script is using default learning rate

### ISSUE THREAD

**Issue by dylanil** [mw281685/IFoA-FL-WP#9]
This needs to use the learning rate found from the skorch tuning as the hyperparameters found from there correspond to the specific learning rate

**Comment by mw281685** [comment]
fixed with commit adb869a
